app/helpers/admin_helper.rb

```diff
@@ -103,4 +103,11 @@ def admin_participant_role_name(class_name)
       "unknown"
     end
   end
+
+  def training_statuses
```

Comment:
I think we should also list "active"? Otherwise once you’ve set them as deferred or withdrawn, there’s no way to undo and make them active again?

Reply:
That was considered by Benedict when we discussed the ticket and he discarded it

Reply:
SITs can re-register a participant withdrawn or deferred and won't contact support for that, he says

Reply:
Hmm really? I’d have thought it’d be better for the same participant profile to get put back into active status after someone defers (or if they un-withdraw) rather than creating a new one?

Reply:
Well, internally the service doesn't create a new one but modifies the existing record in the db

Reply:
ah ok! Fair.